Containerfile

```diff
@@ -1,8 +1,11 @@
 # Provide specific arg for setting the version of nodejs to use.
 # Should match what's in flake.nix for development.
 ARG NODE_MAJOR_VERSION=20
+ARG PNPM_VERSION=10.2.0
 FROM docker.io/node:${NODE_MAJOR_VERSION}-alpine AS base
-RUN corepack enable pnpm
+# We no longer use `corepack enable pnpm` due to breakage documented in
+# https://github.com/nodejs/corepack/issues/612
+RUN npm install -g pnpm@${PNPM_VERSION}
```

Comment:
The substantive change is the removal of `corepack` and use of `npm install pnpm` instead.